### ISSUE TITLE
IRIS: Isis exchange gas (IEG)

### ISSUE THREAD

**Issue by AdrianPotter** [ISISComputingGroup/IBEX#1577]
Iris currently uses an exchange gas device (see NDXIRIS: `C:\LabVIEW Modules\Drivers\ISIS Exchange Gas`). It's in the process of being rebuilt, so not needed for cycle 16-3, but Iris will eventually need an IOC/OPI for it

**Comment by kjwoodsISIS** [comment]
Some questions we need to answer before we can do this ticket:

1. which cycle is IEG support needed for?
   1. 2016/04, starting 15-Nov-16
   1. 2016/05, starting 14-Feb-17
   1. or later?
1. Will the re-built IEG have the same interface as the original IEG? (i.e. would the current LabVIEW module still be able to drive the re-built IEG?)
1. How complex is the IEG (is a it a single device with a simple comms protocol, or is it a composite of multiple devices (c.f. the 3D Magnet).  Should we write our own IOC(s) or use lvDCOM to provide an EPICS wrapper to the current LabVIEW module?

**Comment by kjwoodsISIS** [comment]
Now superseded by #2469.
